_conferences/eurosys2022/results.md

```diff
@@ -255,7 +259,7 @@ artifacts:
       </td>
       <td>
         {% if artifact.award %}
-          <b>Distinguished&nbsp;Artifact</b><br>
+          <b>{{ artifact.award }}</b><br>

```

Comment:
This needs a matching change in the _result-header for summaries (and remove "award_name" from the overall data, I guess)

Reply:
Thanks, good catch! Done.